backend/tools/scrape.py

```
@@ -0,0 +1,358 @@
+        shoeSizeOption(int): 검색할 신발 사이즈. 미지정 시 무시됩니다. (220mm 이하: 220, 290mm 이상: 290)
+                사용 가능한 값 : "220","225","230","235","240","245","250","255","260","265","270","275","280","285","290"
+    
+    Returns:
+        Dict[str, Any]: 검색 결과를 담은 딕셔너리. 반환되는 상품 정보는 LLM이 사용자에게 답변하기 용이하도록 'goodsName', 'brandName', 'price, 'saleRate' 등 핵심 필드만으로 정제되어 있습니다.
+    """
+    print(f"검색 시작 - 키워드: {keyword}, 조건: gender={gender}, price={minPrice}-{maxPrice}")
+    
+    # 1. 기본 상품 검색
+    search_result = search_basic_products_impl(keyword, gender, minPrice, maxPrice, color, shoeSize)
+    print(f"검색 완료 by search_result : {search_result}")
+    if search_result.get('success') != True:
```

Ruff (0.12.2)

Avoid inequality comparisons to `True`; use `not search_result.get('success'):` for false checks

Replace with `not search_result.get('success')`

(E712)